pkg/controllers/execution/execution_controller.go

```diff
@@ -154,31 +155,43 @@ func (c *Controller) syncToClusters(cluster *v1alpha1.Cluster, work *workv1alpha
 		return err
 	}
 
+	var errs []error
 	for _, manifest := range work.Spec.Workload.Manifests {
 		workload := &unstructured.Unstructured{}
 		err := workload.UnmarshalJSON(manifest.Raw)
 		if err != nil {
 			klog.Errorf("failed to unmarshal workload, error is: %v", err)
-			return err
+			errs = append(errs, err)
+			continue
 		}
 
 		applied := helper.IsResourceApplied(&work.Status)
 		if applied {
 			err = c.tryUpdateWorkload(cluster, workload, clusterDynamicClient)
 			if err != nil {
 				klog.Errorf("Failed to update resource in the given member cluster %s, err is %v", cluster.Name, err)
-				return err
+				errs = append(errs, err)
 			}
 		} else {
 			err = c.tryCreateWorkload(cluster, workload)
 			if err != nil {
 				klog.Errorf("Failed to create resource in the given member cluster %s, err is %v", cluster.Name, err)
-				return err
+				errs = append(errs, err)
```

Comment:
I'm ok to set `appliedFailed` in case of part of `manifest` failed but should be given clear info in the condition messages.

Reply:
E.g. add (<succeed>/<totoal>) to the messages.